src/ov_utils.cpp

```diff
@@ -0,0 +1,58 @@
+//*****************************************************************************
+// Copyright 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#include "ov_utils.hpp"
+
+#include <memory>
+
+#include <inference_engine.hpp>
+#include <spdlog/spdlog.h>
+
+namespace ovms {
+
+Status createSharedBlob(InferenceEngine::Blob::Ptr& destinationBlob, InferenceEngine::TensorDesc tensorDesc) {
+    try {
+        switch (tensorDesc.getPrecision()) {
+        case InferenceEngine::Precision::FP32:
```

Comment:
How about U16 and FP16?

Reply:
This is just code move from ov_utils.hpp. It seems that those precisions were absent there from start.

Reply:
I copied handling from deserialization.hpp.